source/modules/collection-api/address/overview.md.erb

```diff
@@ -12,15 +12,15 @@
 }
 ```
 
-Numerous resources have associated location data: Sale payments have an associated billing address, events occur at a particular address, or an account may have a shipping address associated with it. Inspetor standardizes the format in which we collect this data as follows:
+Numerous resources have associated location data: Order payments have an associated billing address, deliveries have the delivery address, or an user may have an address associated with it. Inspetor standardizes the format in which we collect this data as follows:
 
 Property          | Required? | Type   | Description
 --------          | --------- | -----  | -----------
 street_name       | Yes       | String | The street of the address
 street_number     | Yes       | String | The street number of the address
 zip_code          | Yes       | String | The zip code (*e.g. CEP*) of the address
 city              | Yes       | String | The city in which the address is located
-state             | Yes       | String | The two-character ISO 3166-2 abbreviation of the state in which the address is located
+state             | Yes       | String | The up to three-character ISO 3166-2 abbreviation of the state in which the address is located
```

Comment:
Hmm so [it seems](https://en.wikipedia.org/wiki/ISO_3166-2) that the first part of the code is always 2 characters, and the second part, which can indeed be up to 3 characters I think we don't care about

Reply:
my b, first part of ISO 3166-2 is country code, second part state